examples/tutorials/Protein_Generation.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<p>Make sure to add standard deepchem header</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4303/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Added</p>